packages/playwright-core/src/server/electron/electron.ts

```diff
@@ -134,12 +135,26 @@ export class Electron extends SdkObject {
 
       const browserLogsCollector = new RecentLogsCollector();
       const env = options.env ? envArrayToObject(options.env) : process.env;
+
+      let command: string;
+      try {
+        command = options.executablePath || require('electron/index.js');
+      } catch (error: any) {
+        if ((error as NodeJS.ErrnoException).code === 'MODULE_NOT_FOUND') {
```

Comment:
These are two distinct cases with two different errors: with and without the option. Let's report them separately.